test/e2e/data/infrastructure-provider-byoh/v1beta1/cluster-template.yaml

```diff
@@ -183,6 +183,8 @@ kind: ByoCluster
 metadata:
   name: ${CLUSTER_NAME}
 spec:
+  bundleLookupBaseRegistry: projects.registry.vmware.com/cluster_api_provider_bringyourownhost
+  bundleLookupTag: v0.1.0_alpha.2
```

Comment:
Is this going to be updated with every release, if yes then I think its better to use it from variable?

Reply:
@shamsher31 It is not going to change frequently for a bundle of specific k8s version/OS, But I agree with what you suggested. Let others provide their opinion as well.

Reply:
I'm okay with this for short-term. But for longer term, IMO, this field should be optional and if not provided should be treated as `latest` value. Not very sure how tagging works. Does this mean we will have 2 tags for the same image? Is that even allowed?

Otherwise, makes sense to read from a variable.
Do we open a new issue for this and discuss options there? This PR is good to go! (after we have the artifact in the registry, of course!)

Reply:
Created this Issue #259 to decide on the other options.